README.md

```diff
@@ -20,6 +20,18 @@ custom:
         collectorUrl: Paste your url from step 1. here
 
         # Optional, default pattern is "[timestamp=*Z, request_id=\"*-*\", event]"
+        # NOTE: Some runtimes report a log level as the first parameter, so the
+        #       default will not report any of your logs. You'll need to define
+        #       a `filterPattern` that matches the behavior of your runtime.
+        #
+        #       Most of the time, you can add `log_level` as the first parameter.
+        #       (e.g. "[log_level, timestamp = *Z, request_id, event]")
```

Comment:
for me, the `request_id` default of `\"*-*\"` wasn't working, so I pasted the solution that worked for me.